### ISSUE TITLE
Background is not applied after a 2nd color is applied

### ISSUE THREAD

**Issue by antar4** [silkfire/Pastel#37]
If you apply color to two or more strings and then apply background color to the entire line, the background color is displayed ONLY in the first string and empty space

Reproduce:
```
            // Setup
            string t1 = "HELLO ";
            string t2 = "WORLD";

            t1 = t1.Pastel(Color.Yellow);
            t2 = t2.Pastel(Color.Red);

            // Execution 1
            StringBuilder sb = new StringBuilder();
            sb.Append(t1);
            sb.Append(t2);
            string line = sb.ToString().PadRight(110);
            line = line.PastelBg(Color.Blue);

            Console.WriteLine(line);

            // Execution 2
            string t = t1 + t2;
            t = t.PadRight(110);
            t = t.PastelBg(Color.Blue);
            Console.WriteLine(t);
```

Result:
![image](https://github.com/user-attachments/assets/773f7a7e-2ba4-44c5-be9e-34e67cd89f4e)

Expected:
The word "WORLD" should also have a blue background.

Note:
This behavior is new, previous version was working as intended.

**Comment by silkfire** [comment]
Thanks for reporting the issue. I'll take a look at it as soon as I have time.

**Comment by silkfire** [comment]
@antar4 I've published a pre-release version including a fix. Could you try it out and verify that it works for you?

https://www.nuget.org/packages/Pastel/6.0.1-alpha

**Comment by antar4** [comment]
Hi, for my case works perfectly as it did before. Thanks

**Comment by silkfire** [comment]
Fixed in [release v6.0.1](https://github.com/silkfire/Pastel/releases/tag/v6.0.1)
